src/pcapi/core/search/backends/algolia.py

```diff
@@ -29,6 +32,41 @@
 DEFAULT_LONGITUDE = 2.409289
 DEFAULT_LATITUDE = 47.158459
 
+WORD_SPLITTER = re.compile(r"\W+")
+
+
+def url_path(url):
+    """Return the path component of a URL.
+
+    Example::
+
+        >>> url_path("https://example.com/foo/bar/baz?a=1")
+        "/foo/bar/baz?a=1"
+    """
+    if not url:
+        return None
+    parts = urllib.parse.urlparse(url)
+    path = parts.path
+    if parts.query:
+        path += f"?{parts.query}"
+    if parts.fragment:
+        path += f"#{parts.fragment}"
+    return path
+
+
+def remove_stopwords(s: str) -> str:
+    """Remove French stopwords from the given string and return what's
+    left, lowercased.
+
+    We are not interested in being thorough. App Search takes care of
+    ignoring stopwords and does it better than we do. Here we are
+    mostly interested in storing less data in App Search. But we want
+    to keep the order in which words appear (because it matters in App
+    Search).
```

Comment:
Hu hu, le copier-coller ;)

```suggestion
    We are not interested in being thorough. Algolia takes care of
    ignoring stopwords and does it better than we do. Here we are
    mostly interested in storing less data in Algolia. But we want
    to keep the order in which words appear (because it matters in 
    Algolia).
```